ui/app/clinical/common/controllers/visitController.js

```diff
@@ -16,22 +16,19 @@ angular.module('bahmni.clinical')
             $scope.patientUuid = $stateParams.patientUuid;
             $scope.visitUuid = $stateParams.visitUuid;
             var tab = $stateParams.tab;
-            $scope.$on('observations.providers', function (event, observations) {
-                var consultationObservations = observations.filter(function (obs) {
-                    return obs.encounterTypeName === 'Consultation';
-                });
+            var encounterType = appService.getAppDescriptor().getConfigValue("encounterAttributes").encounterType;
```

Comment:
What would a sample config look like? 
- name should be well understood - maybe - 'filterEncounterTypes'?
- should be an array and is optional, maybe non-existent. 
- the app can by default scan all encounter type or 'consultation' encounterType if not configured in app-config